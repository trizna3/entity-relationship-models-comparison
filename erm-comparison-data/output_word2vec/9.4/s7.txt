vehicle-vehicle
line-line
vehicle type-vehicle types
stop-duration
REBIND_MN_TO_1NN1:S-line-stop
MOVE_ATTR_TO_INCIDENT_ASSOCIATION:S-time-(duration(MANY),line(ONE))
CHANGE_CARDINALITY:S-route-line
CHANGE_CARDINALITY:S-line-route
REBIND_1NN1_TO_MN:S-route
CHANGE_CARDINALITY:S-stop-duration
CHANGE_CARDINALITY:S-duration-stop
CHANGE_CARDINALITY:S-line-stop
REBIND_1NN1_TO_MN:S-stop
