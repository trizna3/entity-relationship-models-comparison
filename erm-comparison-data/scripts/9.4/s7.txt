entity set stop {
	"name"
	"position"
};

entity set line {
	"number"
};

entity set vehicle {
	"code"
	"schedule"
	"timestamps"
};

entity set "vehicle types" {
	"bus"
	"trolleybus"
	"tram"
};

entity set route {
	"sequence"
	"x coordinate"
	"y coordinate"
};

association between {
	"vehicle types" ONE
	vehicle MANY
}{};

association between {
	"vehicle types" MANY
	line MANY
}{};


association between {
	line MANY
	vehicle ONE
}{};

association "duration" between {
	line MANY
	stop MANY
}{
	"time"
};

association between {
	line MANY
	route ONE
}{};

association between {
	stop ONE
	route MANY
}{};
