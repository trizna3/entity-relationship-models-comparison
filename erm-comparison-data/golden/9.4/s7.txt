vehicle-vehicle
vehicle type-vehicle types
line-line
stop-stop